remoulade/collection_results.py

```diff
@@ -94,9 +99,35 @@ def completed_count(self) -> int:
         # we could use message.completed here but we just want to make 1 call to get_status
         return backend.get_status(self.message_ids)
 
+    @overload
+    def get(
+        self: "CollectionResults[Result[ResultT]]",
+        *,
+        block: bool = False,
+        timeout: Optional[int] = None,
+        raise_on_error: Literal[True] = True,
+        forget: bool = False,
+    ) -> Iterator[ResultT]:
+        ...
+
+    @overload
+    def get(
+        self: "CollectionResults[Result[ResultT]]",
+        *,
+        block: bool = False,
+        timeout: Optional[int] = None,
+        raise_on_error: bool = True,
+        forget: bool = False,
+    ) -> Iterator[Union[ErrorStored, ResultT]]:
+        ...
+
+    @overload
```

Comment:
why this one ?

Reply:
This allows inferring group().get() type